src/app/Fake.Sql.SqlServer/Sql.SqlServer.fs

```diff
@@ -0,0 +1,170 @@
+namespace Fake.Sql
+
+/// Contains helpers around interacting with SQL Server databases.
+[<RequireQualifiedAccess>]
+module SqlServer =
+
+    open System
+    open System.IO
+    open System.Data.SqlClient
+    open Microsoft.SqlServer.Management.Smo
+    open Microsoft.SqlServer.Management.Common
+
+    open Fake.Core
+    open Fake.IO
+    open Fake.IO.FileSystemOperators
+
+    type ServerInfo = {
+        Server: Server
+        ConnBuilder: SqlConnectionStringBuilder }
```

Comment:
Somehow I don't like this mutable design, but if you can persuade this that this improves the API/design or is needed for some reason I'm happy to merge it (we might consider adding that as comment)

Reply:
I ported what FAKE 4 had, and I also do not like it. I will try some things out in a branch to not hold up this PR if that is ok.